.circleci/main_config.yml

```diff
@@ -430,3 +435,17 @@ workflows:
           message: "Detected nonzero diff in rollup config. This PR modifies the configuration of at least one chain in the registry, which may or may not have been intended"
           context: circleci-superchain-registry
           requires: [compute-rollup-config-diff]
+
+  compute-config-diff-and-report-forked-pr:
```

Comment:
Should we drop "report" from the name, because this never does any reporting?

Reply:
yes